components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -971,21 +1090,26 @@ def render_movie (request, iid, axis, pos, conn=None, **kwargs):
             movie = open(fn).read()
             os.close(fo)
             rsp = HttpResponse(movie, content_type=mimetype)
-            rsp['Content-Disposition'] = 'attachment; filename="%s"' % (img.getName()+ext)
+            rsp['Content-Disposition'] = 'attachment; filename="%s"' \
+                % (img.getName()+ext)
             rsp['Content-Length'] = len(movie)
             return rsp
         else:
             fobj.close()
-            #shutil.move(fn, fn + ext)
-            return HttpResponseRedirect(settings.STATIC_URL + 'webgateway/tfiles/' + rpath)#os.path.join(rpath, img.getName() + ext))
+            # shutil.move(fn, fn + ext)
+            return HttpResponseRedirect(settings.STATIC_URL +
+                                        'webgateway/tfiles/' + rpath)
+            # os.path.join(rpath, img.getName() + ext))
```

Comment:
Really not sure about this one, a commented out line of code following a return! Nothing wrong re flake8 here though.